deploy/gatekeeper.yaml

```diff
@@ -350,6 +350,11 @@ spec:
           requests:
             cpu: 100m
             memory: 256Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsGroup: 3000
+          runAsNonRoot: true
+          runAsUser: 2000
```

Comment:
what's the main reason for changing the uid here vs the one in the dockerfile: https://github.com/open-policy-agent/gatekeeper/blob/master/Dockerfile#L18

Reply:
Just to show that we can. Would you prefer them to be identical? I have no strong opinions either way.

Reply:
After deploying this change, I'm getting the following after I exec into the container:
```
groups: cannot find name for group ID 3000
id manager
uid=1000(manager) gid=999(manager) groups=999(manager)
```

Reply:
I'm not sure what the significance of that is or what response you're looking for.

Is your shell broken? Is there loss of functionality? Is it ugly?

Reply:
not a problem with my shell. but rather
1. `runAsGroup: 3000` failed because the group 3000 does not exist in /etc/group. 
2. I think we should be consistent and use the `manager` user with uid 1000 as we have currently in the Dockerfile. Otherwise as you can see from my shell output, there is a `manager` user with uid 1000, but it is not used to run the manager process.  Let's be consistent here if we can.

Reply:
SGTM, so UID=1000 GID=999?

Reply:
SGTM

Reply:
fixed